go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/athenianco/cloud-common
+
+go 1.13
+
+require (
+	github.com/athenianco/metadata v0.0.0-20200619130750-022c72de9d66
```

Comment:
`common` must not depend on the MD. Let's copy everything that is needed to this package to avoid this dependency.

Reply:
done
p.s. when I told that we need to move pubsub you said no